Gopkg.toml

```diff
@@ -3,7 +3,7 @@
 
 [[constraint]]
   name = "github.com/codedellemc/gocsi"
-  revision = "996f4bd357b3f628be6cfc8a5ffcf67f9345b0a1"
+  revision = "00d5dd1afec2c204bf084a552685888c821ed445"
```

Comment:
Hi @codenrhoden,

Please use `branch = "master"` for any references to GoCSI or any other of our shared projects for which there isn't a tag. The only exception is when pointing to a fork. Otherwise what happens is upstream projects cannot import your dependency graph due to a particular dependency (the one above) being unable to participate in constraint resolution.

Reply:
Good catch. I'm pushed a new revision.